app/scripts/views/base.js

```diff
@@ -762,6 +762,13 @@ define(function (require, exports, module) {
       this._hasNavigated = true;
     },
 
+    navigateViaServer (url) {
```

Comment:
How about `request` vs `navigate`? Does that better indicate that one of them is going to make an HTTP request?